README_PTBR.md

```diff
@@ -0,0 +1,40 @@
+<div align="center">
+    <h3>Hi 👋, I'm <a href="https://longtao.fun">Longtao</a></h3>
```

Comment:
Forgot to translate this part, this is simply
`<h3>Olá 👋, eu me chamo <a href="https://longtao.fun">Longtao</a></h3>`